lib/awspec/helper/finder/backup.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+module Awspec::Helper
+  module Finder
+    module Backup
+      def find_backup_vault(id)
+        res = backup_client.list_backup_vaults
```

Comment:
For example, the following code could be used to achieve pagination.

https://github.com/k1LoW/awspec/blob/ee26a0ee7a724cbf46faccc4b67a557bc073ee97/lib/awspec/helper/finder/acm.rb#L6-L17

ref: https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/Backup/Client.html#list_backup_vaults-instance_method